gourmet/reccard.py

```diff
@@ -1054,7 +1054,7 @@ def revert_cb (self, *args):
     def delete_cb (self, *args):
         self.rg.rec_tree_delete_recs([self.current_rec])
 
-    def close_cb (self, *args):
+    def close_cb(self, *args: Tuple[Gtk.Window, Gdk.Event]) -> bool:
```

Comment:
If we already know that it's only a two-tuple, why `*args`?

Reply:
I didn't change what was already there, and as both values are ignored, I considered it good enough.

Reply:
Okay. Keep it that way. I will change it, if `mypy` complains. :)